src/sage/schemes/elliptic_curves/ell_point.py

```diff
@@ -3982,6 +3998,17 @@ def log(self, base):
             sage: Q.log(P)
             400
 
+        ::
+
+            sage: # needs sage.rings.finite_rings
+            sage: F = GF((5, 60), 'a')
+            sage: E = EllipticCurve(F, [1, 1])
+            sage: E.abelian_group().invariants()
+            (4464, 194301464603136995341424045476456938000)
+            sage: P, Q = E.gens()[::-1]
```

Comment:
Is this reversal because of point order? Might be worth making that clear?

```py
P, Q = E.gens()
if P.order() != 4464:
    P, Q = Q, P
```

or something?